servicemesh-install/topics/install.adoc

```diff
@@ -27,301 +75,566 @@ The 3scale Istio Adapter is deployed and configured in the custom resource file.
 |`300`
 
-|`cacheRefreshSeconds`
+|`PARAM_THREESCALE_CACHE_REFRESH_SECONDS`
 |Time period before expiry when cache elements are attempted to be refreshed
+|Time period in seconds
 |`180`
 
-|`cacheEntriesMax`
+|`PARAM_THREESCALE_CACHE_ENTRIES_MAX`
 |Max number of items that can be stored in the cache at any time. Set to `0` to disable caching
+|Valid number
 |`1000`
 
-|`cacheRefreshRetries`
-|Time period before expiry when cache elements are attempted to be refreshed
+|`PARAM_THREESCALE_CACHE_REFRESH_RETRIES`
+|The number of times unreachable hosts are retried during a cache update loop
+|Valid number
 |`1`
 
-|`AllowInsecureConn`
-|Allow to skip certificate verification when calling `3scale` APIs. Enabling this is not recommended
+|`PARAM_THREESCALE_ALLOW_INSECURE_CONN`
+|Allow to skip certificate verification when calling `3scale` APIs. Enabling this is not recommended.
+|`true`/`false`
 |`false`
 
-|`clientTimeoutSeconds`
+|`PARAM_THREESCALE_CLIENT_TIMEOUT_SECONDS`
 |Sets the number of seconds to wait before terminating requests to 3scale System and Backend
+|Time period in seconds
 |`10`
 |===
 
-[[installing-operator]]
-== Installing the operator
-////
-TODO
-Add an overview of Operators
-////
-The {ProductShortName} installation process introduces a Kubernetes _operator_ to manage the installation of the control plane within the `istio-system` namespace. This operator defines and monitors a custom resource related to the deployment, update, and deletion of the control plane.
-
-You can find the https://github.com/Maistra/openshift-ansible/tree/maistra-0.9/istio[operator templates on GitHub].
-
-[NOTE]
-====
-You *must* name the custom resource `istio-installation`, that is, the metadata value for `name` must be `istio-installation` and you *must* install it into the `istio-operator` namespace that is created by the operator.
-====
-
-The following commands install the {ProductShortName} operator into an existing {product-title} installation; you can run them from any host with access to the cluster. Ensure that you are logged in as a cluster admin before executing these commands.
-
-----
-$ oc new-project istio-operator
-$ oc new-app -f istio_product_operator_template.yaml --param=OPENSHIFT_ISTIO_MASTER_PUBLIC_URL=<master public url>
-----
-
-[NOTE]
-====
-The OpenShift Master Public URL must be configured to match the public URL of your {product-title} Console, this parameter is required by the Fabric8 Launcher.
-====
 
 [[update-mixer-policy-enforcement]]
 == Update Mixer policy enforcement
-In previous versions of {ProductName}, Mixer’s policy enforcement was enabled by default. However, starting with {ProductName} {ProductVersion} Mixer policy enforcement is now disabled by default. You must enable it before running policy tasks.
+In previous versions of {ProductName}, Mixer’s policy enforcement was enabled by default. Mixer policy enforcement is now disabled by default. You must enable it before running policy tasks.
 
```

Comment:
I believe this section is not in the right place. We are instructing the user to update a thing that is not even installed (We're going to install only in the next section). I believe we should move that to any place below installation section

Reply:
In order for what is installed to work, the Mixer policy enforcement status must be enabled. I think it makes more sense to do it _before_ the install, otherwise, people will forget to do it and report issues.